clients/java/src/main/java/io/zeebe/client/impl/subscription/JobRunnableFactory.java

```diff
@@ -47,10 +47,14 @@ private void executeJob(ActivatedJob job, Runnable doneCallback) {
           job.getKey(),
           job.getType(),
           e);
+      String message = e.getMessage();
+      if (message == null) {
+        message = "Unknown error";
```

Comment:
What do you think of `e.getClass().getName()` instead of the generic message. Then we at least see what kind of exception was thrown.

Reply:
What about sending the stacktrace instead, then you have at least a link to your code? A NPE could happen everywhere.

Reply:
True, didn't we planned to add this as a separate property?

Reply:
At the moment we decided to only have a single message field which content the user can choose. So if we want the stacktrace it will also go in to the message property.

Reply:
Wouldn't that mean that the entire stack trace would show up as the Incident name in Operate?

Reply:
Yes the full stacktrace would be available to operate. In the list UI the message is shortened so it does not break the layout. A user can then expand that message to see the full stacktrace. My concern is with a default error message the user has no clue where the problem exists.

If you do not want to send the stacktrace and instead keep the default error message we should log the stacktrace in the client library to allow a user to investigate the problem.

Reply:
I think we should do both - put the stack trace in Operate, and log it on the client.

Reply:
Ok then please update your PR

Reply:
Updated.